### ISSUE TITLE
Connect from internet

### ISSUE THREAD

**Issue by cdelgadob** [cfn-modules/rds-aurora-serverless-postgres#2]
Hello,

I just deployed a serverless cluster but I can't find a way to connect from outside.

Can you please point me in the right direction?
In case it helps, in my deployment I didn't specify any hosted zone, as my DNS are managed outside amazon.

Thanks

**Comment by michaelwittig** [comment]
What's your reason to connect to the Aurora database?

It is not possible to expose an Aurora Serverless to the Internet. You have to spin up an EC2 Instance in a public subnet to connect to the Aurora database.

**Comment by cdelgadob** [comment]
So I can manipulate the DB  easily while developing and to perform maintenance tasks.

I have created a Cloud9 instance in the same VPC as the cluster, in a public subnet. Then I have granted In the security group of the private subnet where the cluster lives a new inbound rule to allow all traffic coming from the CIDR of the public network.

The Cloud9 instance doesn't have the PostgreSQL tools installed, so running this will install all needed: ```sudo yum install postgresql postgresql-server postgresql-devel postgresql-contrib postgresql-docs```
I think that maybe some packages are not needed, but I didn't have time to clean the list.

Obviously I haven't made all this up by myself, my main guidance has been this tutorial: https://aws.amazon.com/es/getting-started/tutorials/configure-connect-serverless-mysql-database-aurora/

Hope it helps others
